engine/config/1.0/schema.json

```diff
@@ -86,8 +86,8 @@
           "description": "An optional map of keys to extract from the response body. See: https://docs.arcade.dev/home/auth-providers/oauth2#jsonpath-expressions-in-response_map"
         }
       },
-      "required": ["endpoint", "parameters"],
-      "additionalProperties": false
+      "required": ["endpoint"],
+      "additionalProperties": true
```

Comment:
Removing `params` from the required list, because some instances of `oauthRequestConfig` don't need `params`. (in particular, `user_info_request`)

Reply:
Allowing additional properties here because `user_info_request` needs to tack on `triggers`